spec/have_tag_spec.rb

```diff
@@ -669,4 +669,16 @@
       end
     end
   end
+
+  context 'html and body elements' do
+    asset 'document'
+
+    it 'should find the html element' do
+      expect(rendered).to have_tag('html')
+    end
+
+    it 'should find the body element' do
+      expect(rendered).to have_tag('body')
```

Comment:
I don't quite understand, the ArgumentError is telling me to add additional options. and it works, I have added `with: {...}`
but your spec don't have any ?!?

Reply:
it's a good point, that's because this code is not fully relevant (as original author was proposing restoring functionality), it was consumed and changed, recent version contains this part changed a bit